content/03.methods.md

```diff
@@ -11,10 +11,14 @@
 2. Describe metadata statistics on PubMed central (number of articles, how many articles were processed
 
 ### Comparing Corpora
-1. Spacy to process text via - Lemmatization, removal of stop words
-2. Describe counting frequencies of each lemma
-3. Describe using chi-square test
-4. Describe how to calculate the likelihood and log odds ratio
+We used gensim [@raw:rehurek_lrec] (version 3.8.1) to perform the above preprocessing tasks on the bioRxiv and PubMed Central corpora.

```

Comment:
Did something get lost here? I'm not sure what was above.

Reply:
I initially wrote in the methods section that I used the spaCy package for pre-processing. This wasn't the case as I mixed up this project with another project that consisted of inserting Pubtator Central into a postgres database (in preparation of turning around the text_mining_hetnet_manuscript). 

The changed part here is that the annorxiver project only uses the gensim package to remove stop words and I _didn't_ perform any lemmatization (converting words into their root form).